hack/test-cmd.sh

```diff
@@ -98,6 +98,17 @@ export OPENSHIFT_PROFILE="${CLI_PROFILE-}"
 # Begin tests
 #
 
+osc get templates
+osc create -f examples/sample-app/application-template-dockerbuild.json
```

Comment:
Create shouldn't create the things in the template anyway (otherwise the template isn't providing value) but yes, I agree.